### PR TITLE
Add Stable Diffusion 2.0 support for SD Finetuning Workflow

### PR DESCRIPTION
**Pull request by harubaru**

### CODE REVIEW

**Review by wbrown — CHANGES_REQUESTED** [review]
Bug to fix.

File: sd-finetuner-workflow/sd-finetuner/requirements.txt [hunk 1]
```diff
@@ -1,4 +1,4 @@
-diffusers>=0.9.0
+git+https://github.com/huggingface/diffusers/
```
Comment (wbrown) [inline sd-finetuner-workflow/sd-finetuner/requirements.txt:1]: This should really be pinned to a hash, instead of accepting whatever master is.

**Review by wbrown — CHANGES_REQUESTED** [review]
Add a hash to the diffusers checkout. We don't want this breaking because of an update to master.

File: sd-finetuner-workflow/sd-finetuner/requirements.txt (hunk 1 above)
Comment (wbrown) [reply sd-finetuner-workflow/sd-finetuner/requirements.txt:1]: Also, you should be teliing it what the egg name will be.

i.e. the following example shows using a checkpoint + egg: `git+https://git@github.com/Dango233/diffusers.git@dango.dev.nov_update#egg=diffusers`

Comment (harubaru) [reply sd-finetuner-workflow/sd-finetuner/requirements.txt]: Got it pointing to this commit which implements v-prediction: https://github.com/huggingface/diffusers/commit/6c56f05097f7d3c561f02dc1c27e3dd7e9f88ce1

**Pull request merged**